test/tests/nodalkernels/constraint_enforcement/interpolated-ncp-lm-nodal-enforcement-nodal-forces.i

```diff
@@ -0,0 +1,122 @@
+l=10
+nx=100
+num_steps=10
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  xmax = ${l}
+  nx = ${nx}
+[]
+
+[Variables]
+  [u]
+  []
+  [lm]
+  []
+[]
+
+[ICs]
+  [u]
+    type = FunctionIC
+    variable = u
+    function = '${l} - x'
+  []
+[]
+
+[Kernels]
+  [time]
+    type = TimeDerivative
+    variable = u
+  []
+  [diff]
+    type = Diffusion
+    variable = u
+  []
+  [ffn]
+    type = BodyForce
+    variable = u
+    function = '-1'
+  []
+[]
+
+[NodalKernels]
+  [positive_constraint]
+    type = LowerBoundNodalKernel
+    variable = lm
+    v = u
+    exclude_boundaries = 'left right'
+  []
+  [forces]
+    type = CoupledForceNodalKernel
+    variable = u
+    v = lm
+  []
+[]
+
+
+[BCs]
+  [left]
+    type = DirichletBC
+    boundary = left
+    value = ${l}
+    variable = u
+  []
+  [right]
+    type = DirichletBC
+    boundary = right
+    value = 0
+    variable = u
+  []
+[]
+
+[NodalKernels]
+[]
+
+[Preconditioning]
+  [smp]
+    type = SMP
+    full = true
+  []
+[]
+
+[Executioner]
+  type = Transient
+  num_steps = ${num_steps}
+  solve_type = NEWTON
+  dtmin = 1
+  petsc_options_iname = '-snes_max_linear_solve_fail -ksp_max_it -pc_type -sub_pc_factor_levels -snes_linesearch_type'
+  petsc_options_value = '0                           30          asm      16                    basic'
```

Comment:
The default is not zero for `snes_max_linear_solve_fail`?

Reply:
No it's not. MOOSE just lets bad linear solves go on forever; it relies on the non-linear solve to not converge